_projects/0400_blue-button-2.md

```diff
@@ -6,15 +6,15 @@ banner_file_mobile: banner--people-projects-sm.svg
 project_page: true
 
 # Carousel (Edit this)
-carousel_title: "Modernizing health data access with Medicare APIs"
-carousel_summary: "We worked with CMS to build an API that enables beneficiaries with different needs to grant access to developers that can help them monitor for drug conflicts, refill prescriptions, and track progress towards desired healthcare outcomes."
-carousel_image_name: project-hhs-blue-button-card.jpg
+carousel_title: "Empowering beneficiares with their own healthcare data"
+carousel_summary: "USDS has worked with CMS to empower beneficiaries with their own health data. Leveraging CMS APIs, providers can view of their patients’ full healthcare history without patient intervention. Providers save valuable time that they can better spend talking to patients and preventing conflicts or gaps in care."
```

Comment:
noticed this difference btwn the wording in `carousel_title` vs `carousel_summary`:
"...their own healthcare data" vs  "...their own health data". Would it be appropriate to use the same phrase?